apollo-router/src/plugins/telemetry/tracing/datadog/span_processor.rs

```diff
@@ -36,6 +37,7 @@ impl<T: SpanProcessor> SpanProcessor for DatadogSpanProcessor<T> {
             span.span_context.is_remote(),
             span.span_context.trace_state().clone(),
         );
+        span.attributes.push(KeyValue::new("_dd.measured", 1));
```

Comment:
This will set _dd.measured on ALL spans?  If so, do we really want that, or just want certain subset of spans to be measured?

REF: https://github.com/DataDog/documentation/issues/22774#issuecomment-2075965455
> As for measured, that is intentionally undocumented as it must be used carefully, typically with help from our Support team. It allows span metrics to be generated for custom traces.

Reply:
@theJC this is a good callout.  we're going to revert this PR and for the moment, this configuration can be used:

```yaml
   supergraph:
        attributes:
          "_dd.measured":
            static: 1
```